scapy/arch/windows/__init__.py

```diff
@@ -626,19 +635,21 @@ def _ask_user():
             # No action needed
             return
         else:
-            warning(
+            log_interactive.warning(
                 "Scapy has detected that your pcap service is not running !"
             )
             if not conf.interactive or _ask_user():
                 succeed = pcap_service_start(askadmin=conf.interactive)
                 if succeed:
                     log_loading.info("Pcap service started !")
                     return
-        warning("Could not start the pcap service ! "
-                "You probably won't be able to send packets. "
-                "Deactivating unneeded interfaces and restarting "
-                "Scapy might help. Check your winpcap/npcap installation "
-                "and access rights.")
+        log_loading.warning(
+            "Could not start the pcap service ! "
+            "You probably won't be able to send packets. "
+            "Deactivating unneeded interfaces and restarting "
```

Comment:
Shall the user deactivate unneeded interface? If correct, I think that `Please deactivate unneeded interface` is a better message.

Reply:
I actually don't think this is useful at all. I'll just remove it